package.json

```diff
@@ -30,27 +30,36 @@
   "homepage": "http://fluidproject.org",
   "dependencies": {
     "es6-template-strings": "2.0.0",
-    "extend": "^3.0.0"
+    "extend": "3.0.0"
   },
   "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-contrib-jshint": "~0.9.0",
+    "grunt": "1.0.1",
+    "eslint-config-fluid": "1.0.0",
+    "fluid-grunt-eslint": "18.1.2",
     "grunt-jsonlint": "1.0.4",
-    "sinon": "^1.17.1"
+    "sinon": "1.17.1"
   },
   "defaultOptions": {
```

Comment:
Yes, the default values aren't easily modifiable. In execution they are, but I agree that's not great for reuse and building upon.